package.json

```diff
@@ -61,8 +63,11 @@
     "core-js": "^2.6.3",
     "cpx": "^1.5.0",
     "cross-env": "^5.2.0",
+    "cypress": "^3.2.0",
+    "cypress-cucumber-preprocessor": "^1.11.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "1.7.1",
+    "eslint-plugin-cypress": "^2.0.1",
```

Comment:
the package-lock should probably have updated after you ran `npm install` -  can you commit that as well please? Make sure you're on at least version `6.8.0` of `npm` first otherwise you will get some discrepancies.

Reply:
package-lock.json commited after npm install (6.8.0)